cfn-resources/cluster/test/README.md

```diff
@@ -0,0 +1,29 @@
+# Cluster 
+
+## Impact 
+The following components use this resource and are potentially impacted by any changes. They should also be validated to ensure the changes do not cause a regression.
+- Cluster L1 CDK constructor
+- AtlasBasic L3 CDK constructor
+- Encryption at Rest L3 CDK constructor
+- Atlas Quickstart
+- Atlas Quickstart Fargate
+
+
+
+
```

Comment:
```suggestion

```

[q] why we need so many empty lines here and in the readme below (Impact) ? could we stick to a same amount in both?